packages/api-gateway/src/drivers/http/adapters/users/handlers.js

```diff
@@ -0,0 +1,59 @@
+function errorHandler(error, reply) {
+  if (error.isBoom) {
+    return reply.send(error.output.payload);
```

Comment:
Can you add the status code? 
```suggestion
    return reply.code(400).send(error.output.payload);
```

Reply:
I applied ```return reply.code(error.output.statusCode).send(error.output.payload);``` to make the code boom.